src/app/ui-kit/ui-kit.module.ts

```diff
@@ -5,10 +5,12 @@ import { CommonModule } from '@angular/common';
 import { HttpClientModule } from '@angular/common/http';
 import { AngularSvgIconModule } from 'angular-svg-icon';
 import { HeaderComponent } from './layout/header/header.component';
+import { LinkComponent } from './components/link/link.component';
+import { RouterModule } from '@angular/router';
 
 @NgModule({
-  declarations: [ ButtonComponent, TypographyComponent, HeaderComponent ],
-  exports: [ ButtonComponent, TypographyComponent, HeaderComponent ],
-  imports: [ CommonModule, HttpClientModule, AngularSvgIconModule.forRoot() ],
+  declarations: [ ButtonComponent, TypographyComponent, LinkComponent, HeaderComponent ],
+  exports: [ ButtonComponent, TypographyComponent, LinkComponent, HeaderComponent ],
+  imports: [ CommonModule, HttpClientModule, AngularSvgIconModule.forRoot(), RouterModule ],
```

Comment:
It's better to import HttpClientModule on a root level, because it may cause some problems in future